dbUtils.js

```diff
@@ -15,68 +16,191 @@ const getPolls = async () => {
 
 const createPoll = async (request) => {
     const {
-        pollTitle, choices, isClosed, channelName, channelId, teamName, teamId, isGame
+        pollTitle,
+        isClosed,
+        channelName,
+        channelId,
+        teamName,
+        teamId,
+        isGame,
+        choices
     } = request;
-    return db.query('INSERT INTO public.polls (poll_title, choices, is_closed, channel_name, channel_id, team_name, team_id, is_game)' +
+    return db.query('INSERT INTO public.polls (poll_title, is_closed, channel_name, channel_id, team_name, team_id, is_game, choices)' +
```

Comment:
I believe you can use `returning id` or `returning *` at the end of the query on line 28 instead of calling "findOne" to get that information. This would mean you wouldn't need lines 31-34